### PR TITLE
[wasm][xharness] install development SSL certificate on Helix agent before xharness run via SDK or powershell

### LINKED ISSUE #53207: Addition of "dotnet dev-certs" command makes all WASM work items time out
See https://github.com/dotnet/runtime/pull/53180, then most [any log file produced for WASM testing thereafter on windows docker containers](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-53184-merge-94442eaa4fa14f84ac/System.Runtime.CompilerServices.VisualC.Tests/console.877ffb82.log?sv=2019-07-07&se=2021-06-13T22%3A19%3A03Z&sr=c&sp=rl&sig=VJIB40ATEO2xsoWwRgFv7KGPaZtIU0ixiQil2eSY24k%3D):

```
...
C:\helix\work\workitem>dotnet dev-certs https 

...
[EXECUTION TIMED OUT]
Exit Code:-3Executor timed out after 1800 seconds and was killed
...
```

Every WASM work item has timed out at 30 or 45 minutes since, and the queue is thus buried.

This commit should be immediately reverted, then tested locally using `mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-2004-helix-webassembly-amd64-20210422161149-94a50b5`.  

@danmoseley FYI.

### PR DESCRIPTION
**Pull request by pavelsavara**
Caused by https://github.com/dotnet/runtime/pull/53180, https://github.com/dotnet/runtime/pull/53225
The original approach to install certificates didn't work because we don't have dotnet SDK, just runtime on some helix agents.
Fixes https://github.com/dotnet/runtime/issues/53207

### CODE REVIEW

File: src/libraries/sendtohelixhelp.proj [hunk 1]
```diff
@@ -157,21 +148,36 @@
...
-    <IncludeHelixCorrelationPayload Condition="'$(HelixCorrelationPayload)' != '' and '$(TargetOS)' != 'Browser'">true</IncludeHelixCorrelationPayload>
-  </PropertyGroup>
+    <!-- 
+      We are hosting the payloads for the WASM/browser on kestrel in the xharness process.
+      We also run some network tests to this server and so, we are running it on both HTTP and HTTPS.
+      For the HTTPS endpoint we need development SSL certificate.
+    -->
+  <PropertyGroup Condition="''$(TestPackages)' != 'true' and $(TargetsWindows)' != 'true' and '$(BrowserHost)' != 'windows'">
+    <!-- install SDK so that, we could use `dotnet dev-certs https`. -->
+    <IncludeDotNetCli>true</IncludeDotNetCli>
```
Comment (premun) [inline src/libraries/sendtohelixhelp.proj:158]: Will `IncludeDotNetCli` be set for Windows as well?

Comment (premun) [reply src/libraries/sendtohelixhelp.proj:158]: Maybe we can set this one in the bottom with `HelixPreCommands`?

File: src/libraries/sendtohelixhelp.proj [hunk 2]
```diff
@@ -157,21 +148,36 @@
...
   </ItemGroup>
 
-  <PropertyGroup>
-    <HelixPreCommands>@(HelixPreCommand)</HelixPreCommands>
-    <IncludeHelixCorrelationPayload Condition="'$(HelixCorrelationPayload)' != '' and '$(TargetOS)' != 'Browser'">true</IncludeHelixCorrelationPayload>
-  </PropertyGroup>
+    <!-- 
+      We are hosting the payloads for the WASM/browser on kestrel in the xharness process.
+      We also run some network tests to this server and so, we are running it on both HTTP and HTTPS.
+      For the HTTPS endpoint we need development SSL certificate.
```
Comment (premun) [inline src/libraries/sendtohelixhelp.proj:154]:
```suggestion
      We are hosting the payloads for the WASM/browser on Kestrel in the xharness process.
      We also run some network (HTTP and HTTPS) tests that connect to this server.
      In order to set up an HTTPS endpoint we need to install a development SSL certificate.
```

File: src/libraries/sendtohelixhelp.proj (hunk 1 above)
Comment (pavelsavara) [reply src/libraries/sendtohelixhelp.proj:159]: if powershell works for windows, we do not need full SDK.

Comment (premun) [reply src/libraries/sendtohelixhelp.proj:159]: I know but we still need `IncludeDotNetCli`

Comment (premun) [reply src/libraries/sendtohelixhelp.proj:159]: Then we don't need it at all?

**Pull request closed without merging**